src/audiomixerboard.h

```diff
@@ -135,11 +135,7 @@ public slots:
     void OnMuteStateChanged ( int value );
     void OnGroupStateChanged ( int );
 
-    void OnGroupMenuGrpNone() { SetGroupID ( INVALID_INDEX ); }
-    void OnGroupMenuGrp1()    { SetGroupID ( 0 ); }
-    void OnGroupMenuGrp2()    { SetGroupID ( 1 ); }
-    void OnGroupMenuGrp3()    { SetGroupID ( 2 ); }
-    void OnGroupMenuGrp4()    { SetGroupID ( 3 ); }
+    void OnGroupMenuGrp ( int iGrp ) { SetGroupID ( iGrp ); }
```

Comment:
I'm not sure this is needed.  All it's doing is handing off the caught signal to the method.  Nothing outside the class uses this slot, so the `addAction` on the menu could directly call `SetGroupID`.